### ISSUE TITLE
Imported trace displays thread as crashed

### ISSUE THREAD

**Issue by JohanEngelen** [wolfpld/tracy#213]
This bug appeared recently, probably introduced by https://github.com/wolfpld/tracy/pull/211

When importing the test case using `import-chrome`, the thread/process shows up as crashed. (Maybe because it gets thread ID zero?)

Test case:
```json
{
"traceEvents": [
{"ph":"M","ts":0,"args":{"name":"bin/ldc2"},"name":"process_name","pid":101,"tid":101},
{"ph":"M","ts":0,"args":{"name":"bin/ldc2"},"cat":"","name":"thread_name","pid":101,"tid":101},
{"ph":"X","name": "ExecuteCompiler","ts":402,"dur":329467,"loc":"file:123","args":{"detail": "blabla"},"pid":101,"tid":101}
]
}
```

**Comment by wolfpld** [comment]
Yes, this is exactly because TID is 0.

**Comment by wolfpld** [comment]
Fixed in fe71ae32 by changing how PID+TID pairs are encoded to preserve both PID and TID.

![obraz](https://user-images.githubusercontent.com/600573/118358518-7c8cac00-b57f-11eb-8835-0ec488c5e32d.png)

@c-cube
